src/Firehose.Web/Web.config

```diff
@@ -36,10 +36,6 @@
   <system.web>
     <compilation debug="true" targetFramework="4.5.1">
       <assemblies>
-        <add assembly="System.Globalization, Version=4.0.0.0,
```

Comment:
I don't think we should mess with this.

Reply:
Whoops, VS4Mac decided this was a good idea on it's own. Will add it back.